daffodil-runtime1/src/main/scala/org/apache/daffodil/processors/RuntimeData.scala

```diff
@@ -195,8 +194,7 @@ sealed class NonTermRuntimeData(
   @TransientParam pathArg: => String,
   @TransientParam namespacesArg: => NamespaceBinding,
   @TransientParam unqualifiedPathStepPolicyArg: => UnqualifiedPathStepPolicy)
-  extends RuntimeData
-  with PreSerialization {
```

Comment:
Why remove PreSerialization?

Reply:
Moved to base trait. I needed RuntimeData to be serializable.